code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -10,9 +10,15 @@
 		handle_embedded_objects() //Moving with objects stuck in you can cause bad times.
 	if(CE_SPEEDBOOST in chem_effects)
 		tally -= chem_effects[CE_SPEEDBOOST]
-
-
-
+	if(isturf(loc))
+		var/turf/T = loc
+		if(T.get_lumcount() < 0.6)
```

Comment:
Should really check a define

Reply:
which define?

Reply:
One doesn't currently exist for this, maybe we can develop it further in another PR